08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -0,0 +1,154 @@
+* {
+    font-family: sans-serif;
+    color: #606064;
+    font-weight: lighter;
+}
+#page-wrap {
+    width: 800px;
+    margin: 0px auto;
+}
+h1 {
+    color: #46484F;
+    font-size: 30px;
+    text-align: center;
+}
+h3 {
+    font-size: 15px;
+    padding-left: 10px;
+}
+p1 {
+    text-align: center;
+    font-size: 11px;
+    line-height: 20px;
+}
+p {
+    font-size: 11px;
+    line-height: 20px;
+    padding-left: 10px;
+    padding-top: 10px;
+}
+ul {
+    list-style-image: url(http://www.hrandtrainingdoctor.co.uk/wp-content/uploads/2013/07/Green-Tick-e1409164761628.jpg);
+    font-size: 11px;
+    line-height: 20px;
+    padding-left: 20px;
+}
+#keyboard {
+    background-color: #F7F7F7;
+    float: left;
+    width: 350px;
+    height: 300px;
+}
+#keyboard_image {
+    width: 330px;
+    height: 160px;
+    border-top: solid #F7F7F7 10px;
+    border-left: solid #F7F7F7 10px;
+    border-right: solid #F7F7F7 10px;
+}
+aside {
+    float: left;
+    width: 320px;
+    margin-left: 20px;
+}
+.product {
+    background-color: #F7F7F7;
+    height: 77px;
+    padding-top: 10px;
+    margin-bottom: 20px;
+}
+#laptop {
+    float: left;
+    width: 110px;
+    height: 65px;
+    margin-left: 10px;
+    margin-right: 10px;
+}
+#desk {
+    float: left;
+    width: 110px;
+    height: 65px;
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-bottom: 15px;
+}
+#window {
+    float: left;
+    width: 110px;
+    height: 65px;
+    margin-left: 10px;
+    margin-right: 10px;
+}
+h4 {
+    align-items: float: left;
+    font-size: 15px;
+    font-weight: lighter;
+    margin-top: auto;
+}
+h5 {
+    align-items: float: left;
+    padding-left: 20px;
+    font-size: 15px;
+    font-weight: lighter;
+    margin-top: auto;
+}
+h8 {
+    align-items: float: left;
+    padding-left: 10px;
+    font-size: 15px;
+    font-weight: lighter;
+    margin-top: auto;
+}
+p3 {
+    font-size: 11px;
+    font-weight: lighter;
+}
+.clearfix:after {
+    content: ".";
+    visibility: hidden;
+    display: block;
+    height: 20px;
+    ;
+    clear: both;
+}
+footer {
+  width:690px;
+    border-top: #F7F7F7 solid;
+    border-bottom: #F7F7F7 solid;
+    padding-top: 20px;
+    padding-bottom: 140px;
+
+}
+#profession {
+    float: left;
+    border-right: #F7F7F7 solid;
+    width: 228px;
+    height: 120px;
+}
```

Comment:
these three all have the same stylings barring the border. consider using one class and one id to be more efficient